Services/API/Wizzy/WizzyAPIWrapper.php

```diff
@@ -36,6 +36,7 @@ public function __construct(
         $this->authHeaders = $authHeaders;
         $this->wizzyWebhookAPI = $wizzyWebhookAPI;
         $this->storeAdvancedConfig = $storeAdvancedConfig;
+        $this->wizzyAPIEndpoints = $wizzyAPIEndpoints;
     }
 
     public function collectClick(array $clickData, $storeId, array $headers): Response
```

Comment:
Do we need 

$this->storeGeneralConfig->setStore

Before API calls or it is handled?

Reply:
It is handled.